staging/src/k8s.io/csi-translation-lib/translate.go

```diff
@@ -35,9 +35,11 @@ var (
 	}
 )
 
+type CSITranslationLibrary struct{}
```

Comment:
Should it be `interface` instead so individual users don't need to create their own one, as in https://github.com/kubernetes-csi/external-provisioner/pull/343?

(+ `NewCSITranslationLibrary()`  and a private struct as the implementation)